numpy/core/fromnumeric.py

```diff
@@ -837,12 +837,12 @@ def argpartition(a, kth, axis=-1, kind='introselect', order=None):
     return _wrapfunc(a, 'argpartition', kth, axis=axis, kind=kind, order=order)
 
 
-def _sort_dispatcher(a, axis=None, kind=None, order=None):
+def _sort_dispatcher(a, axis=None, kind=None, order=None, by=None):
```

Comment:
If we're following https://github.com/numpy/numpy/pull/16700/files#r460500656 we should also:
```suggestion
def _sort_dispatcher(a, axis=None, kind=None, order=None, *, by=None):
```

Reply:
This may run into #16299, but we should do it anyway.